recipes/cuda-version/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "cuda-version" %}
+{% set version = "12.0.0" %}
+{% set major_version = version.split('.')[0]|int %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  run_constrained:
+    - __cuda >={{ major_version }}
```

Comment:
Curious whether we want the major version constraint here or if we want this more specified

Reply:
This would match what `cudatoolkit` was doing for the versions we would support here:
https://github.com/conda-forge/cudatoolkit-feedstock/blob/main/recipe/meta.yaml#L576-L580
But open to changing as necessary

Reply:
Yeah starting out with matching `cudatoolkit` seems reasonable

Let's see if others have thoughts here

Reply:
This seems reasonable to me. If someone is using newer driver features or something of the like they can always add an explicit constraint directly on `__cuda` if needed.